app/src/main/java/com/adisa/diningplus/MenuAdapter.java

```diff
@@ -63,8 +63,6 @@ public View getChildView(int listPosition, final int expandedListPosition,
         if (item.marked){
             expandedListTextView.setBackgroundColor(context.getResources().getColor(R.color.backgroundMarked));
             expandedListTextView.setTextColor(context.getResources().getColor(R.color.colorMarked));
-        } else {
-            expandedListTextView.setBackgroundColor(context.getResources().getColor(R.color.transparent));
```

Comment:
Have you tested this change? I might have included that line intentionally.

Reply:
It seemed like this was causing some items to be invisible (i.e. white on white) and removing this line fixed the issue. That probably had something to do with my other coloring changes. It seems to work fine but I'll test it more if you like.

Reply:
I tested it this morning, if you load a menu page with a few traits highlighted and scroll up and down, eventually items will be highlighted that do not contain the traits. The else block should be like this:
```suggestion
            expandedListTextView.setBackgroundColor(context.getResources().getColor(R.color.transparent));
            expandedListTextView.setTextColor(context.getResources().getColor(android.R.color.primary_text_light));
```

Reply:
You'll also have to add the transparent background line back in to prevent the unmarked items from having a red background.

Reply:
Done!